doc/api/async_hooks.md

```diff
@@ -0,0 +1,677 @@
+# Async Hooks
+
+> Stability: 1 - Experimental
+
+The `async-hooks` module provides an API to register callbacks tracking the
+lifetime of asynchronous resources created inside a Node.js application.
+It can be accessed using `require('async-hooks')`.
```

Comment:
Also, please move the require statement to it's own line for consistency with other docs. e.g.

```
It can be accessed using:

```js
const async_hooks = require('async_hooks');
```